app/scripts/lib/image_proxy.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore',
+  'jquery',
+  'config'
+], function (_, $, config) {
+  'use strict';
+
+  var EMBEDLY_DISPLAY_ENDPOINT = '//i.embed.ly/1/display';
+
+  // TODO: move this somewhere generic when it's needed in more than one place
+  var IS_RETINA = window.devicePixelRatio >= 1.3;
+
+  var imageProxy = {
```

Comment:
It looks like only the `crop` method is called in this patch. Ordinarily, I'd suggest maybe getting rid of the unused other parts of the API, but this file's whole existence seems a bit awkward/suspect - really seems like the server should be doing the image sizing / proxy URL generation for the front-end. So, since this smells like a short-term fix, I think it's fine as-is.